src/components/vm/consoles/vnc.tsx

```diff
@@ -481,6 +511,8 @@ export class VncActive extends React.Component<VncActiveProps, VncActiveState> {
 
     onSecurityFailure(reason: string | undefined) {
         console.info('Security failure:', reason || "unknown reason");
+        if (reason)
+            this.props.state.setDisconnected(reason);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2038-5a05fe26-20250916-083706-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)